src/features/Navigation/NavigationItems.tsx

```diff
@@ -70,7 +74,7 @@ export const Link = styled(RouterNavLink)`
   }
 `;
 
-const NotificationCircle = styled.div<{ withBorder: boolean }>`
+const UnseenDot = styled.div<{ withBorder: boolean }>`
```

Comment:
Although it's not much code, I would like to see the UnseenDot as its own component now when we're using it in multiple different locations.

Reply:
Here is the ticket https://trello.com/c/Malm41zb/978-refactor-unseendot